src/main/java/komo/fraczek/toukparking/domain/ParkingBill.java

```diff
@@ -0,0 +1,122 @@
+package komo.fraczek.toukparking.domain;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+
+@Entity
+public class ParkingBill {
+
+    @javax.persistence.Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long Id;
+
+    //    Driver type
+    @Enumerated(EnumType.STRING)
+    private DriverType driverType;
+
+    //    Parking activity status
+    @Enumerated(EnumType.STRING)
+    private ParkingStatus parkingStatus;
+
+    //    Vehicle registration number
+    private String numberPlate;
+
+    private int parkingTimeInHours;
+
+    //    Bill issue date
+    private LocalDate date;
+
+    //    Fee grows so fast that after few days it exceedes default DECIMAL(19, 2) range..
+    @Column(name = "PARKING_FEE", columnDefinition = "DECIMAL(30, 2)")
+    private BigDecimal parkingFee;
+
+
+    public ParkingBill() {
+    }
+
+    public ParkingBill(DriverType driverType, String numberPlate) {
+        this.driverType = driverType;
+        this.numberPlate = numberPlate;
+        this.parkingStatus = ParkingStatus.OCCUPIED;
+    }
+
+    public ParkingBill(DriverType driverType, ParkingStatus parkingStatus, String numberPlate, int parkingTimeInHours, LocalDate date, BigDecimal parkingFee) {
+        this.driverType = driverType;
+        this.parkingStatus = parkingStatus;
+        this.numberPlate = numberPlate;
+        this.parkingTimeInHours = parkingTimeInHours;
+        this.date = date;
+        this.parkingFee = parkingFee;
+    }
+
+    public int getParkingTimeInHours() {
```

Comment:
polecam spojrzeć na lomboka

Reply:
Czytałem o projekcie Lombok, jeśli polecasz to pewnie zacznę dodatkowo używać :)